lib/manageiq/appliance_console/cli.rb

```diff
@@ -232,6 +249,23 @@ def set_external_db
       config.post_activation
     end
 
+    def replication
```

Comment:
Can you make this method name something with a verb to match the others (eg. `set_db`, `create_key`, `config_tmp_disk`)? It reads a but strange as is. Maybe `configure_replication`?